### PR TITLE
Add metadata for 21.10.10 models

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

File: metadata/AustralianInvoices__8__metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "inputType": "JSON",
+    "displayName": "AustralianInvoices",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON37_DU",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Document Understanding",
+    "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v21.10",
```
Comment (irinel-birsan-uipath) [inline metadata/AustralianInvoices__8__metadata.json:20]: @vivekjhauipath same min version?

Comment (vivekjhauipath) [reply metadata/AustralianInvoices__8__metadata.json:20]: yes, because these will be part of new baseImage. Though we can potentially use 21.4 too.

**Review by vivekjhauipath — APPROVED** [review]
LGTM

File: metadata/AustralianInvoices__8__metadata.json (hunk 1 above)
Comment (vivekjhauipath) [reply metadata/AustralianInvoices__8__metadata.json:20]: but testing is only valid for 21.10 AIC

**Review by vnaren23 — APPROVED** [review]
LGTM

**Pull request merged**